pkgs/development/compilers/nextpnr/default.nix

```diff
@@ -62,13 +46,13 @@ stdenv.mkDerivation rec {
       --replace 'git log -1 --format=%h' 'echo ${substring 0 11 src.rev}'
   '';
 
-  meta = with stdenv.lib; {
+  dontWrapQtApps = !enableGui;
```

Comment:
If there's binaries in `$out/bin` that don't need to be wrapped we should probably do it manually,

```
postFixup = lib.optionalString (!enableGui) ''
  wrapQtApp $out/bin/$binary
'';

Reply:
All the binaries use Qt when the GUI is enabled (even `nextpnr-generic --gui` pops up an error box), so I think this should be fine.